configs/platforms/sles-11-x86_64.rb

```diff
@@ -19,6 +17,16 @@
     "rsync",
     "zlib-devel"
   ]
+  plat.provision_with(%q{cat <<END > /etc/zypp/repos.d/localmirror-os.repo
+    [localmirror-os]
+    name=localmirror-os
+    baseurl=http://osmirror.delivery.puppetlabs.net/sles-11-sp4-x86_64/RPMS.os
+    enabled=1
+    gpgcheck=0
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-SLES-11
+    autorefresh=1
+    type=rpm-md 
+    END})
```

Comment:
Pasting my note from slack, this unfortunately didn't work for me as the rest of vanagon's provision commands were written into the `localmirror-os.repo` file